packages/auth/src/app.ts

```diff
@@ -215,6 +216,8 @@ export class App {
       grantRoutes.interaction.finish
     )
 
+    this.publicRouter.post('/auth/continue/:id', grantRoutes.post)
```

Comment:
I didn't realize that you reintroduced these "individual" routes (for the lack of a better term) in your previous PR. When you look at lines 167-205, I actually included functionality that only creates routes for those speced out. You should add the one for the `continue` request there (and maybe remove the ones for introspect, rotate, and revoke). Now that we've also speced out the interaction endpoints, we could include them in the functionality of lines 167-205 as well. But that can be in another PR.

Reply:
It actually is already in the openapi routes, and is duplicated in the changed line - I just didn't notice during the rebase so I just need to remove this line.